src/pages/Search/components/RelateSearchItem.tsx

```diff
@@ -1,20 +1,42 @@
 import StoreIcon from '/src/assets/Home/stores-icon.svg';
 import MenuIcon from '/src/assets/Home/menu-icon.svg';
 import NavigateStoreIcon from '/src/assets/Home/navigate-store-icon.svg';
-import { Link } from 'react-router-dom';
+import { Link, useSearchParams } from 'react-router-dom';
+import useLogger from '@/util/hooks/analytics/useLogger';
+import { getCategoryNameById } from '@/constants/shopCategories';
 
 interface RelatedSearchItemProps {
   tag: 'store' | 'menu';
-  content: string;
   to: string;
+  shop_name: string;
+  menu_name?: string;
 }
 
-export default function RelateSearchItem({ tag, content, to }: RelatedSearchItemProps) {
+export default function RelateSearchItem({ tag, shop_name, menu_name, to }: RelatedSearchItemProps) {
+  const logger = useLogger();
+  const [searchParams] = useSearchParams();
+  const categoryParam = searchParams.get('category');
+
+  const handleSearchItemClick = () => {
+    logger.actionEventClick({
+      team: 'BUSINESS',
+      event_label: 'shop_clicked',
+      value: `${shop_name}`,
+      duration_time: (new Date().getTime() - Number(sessionStorage.getItem('selectedCategoryTime'))) / 1000,
+      previous_page: getCategoryNameById(categoryParam ? Number(categoryParam) : undefined),
+      current_page: `${shop_name}`,
+    });
+  };
+
   return (
-    <Link to={to} className="flex h-12 w-full items-center justify-between border-b border-neutral-200 p-2">
+    <Link
+      to={to}
+      className="flex h-12 w-full items-center justify-between border-b border-neutral-200 p-2"
+      onClick={handleSearchItemClick}
+    >
       <div className="flex items-center gap-1">
         {tag === 'store' ? <StoreIcon /> : <MenuIcon />}
-        <p className="text-[10px] text-neutral-500">{content}</p>
+        <p className="text-[10px] text-neutral-500">{menu_name ? `${menu_name} | ${shop_name}` : shop_name}</p>
       </div>
```

Comment:
이런 접근은 어떤 것 같나요? 수정해달라는 의미는 아니고 의견이 궁금해서요!
```suggestion
        <p className="text-[10px] text-neutral-500">
          {menu_name && `${menu_name} | `}
          {shop_name}
        </p>
      </div>
```

Reply:
[ab4e1b4](https://github.com/BCSDLab/KOIN_ORDER_WEBVIEW/pull/214/commits/ab4e1b49de59ddf97c1c200c2a759daa2740acf6)

개인적으로도 말씀해주신 방법이 더 깔끔한 것 같아서 반영해봤습니다!